Formula/phoneinfoga.rb

```diff
@@ -0,0 +1,39 @@
+class Phoneinfoga < Formula
+  desc "Information gathering framework for phone numbers"
+  homepage "https://sundowndev.github.io/phoneinfoga/"
+  url "https://github.com/sundowndev/phoneinfoga/archive/v2.10.2.tar.gz"
+  sha256 "ba6898493c0ac77c178d8942df670cf001caf112d2546560206ea071d6e4dbde"
+  license "GPL-3.0-only"
+  head "https://github.com/sundowndev/phoneinfoga.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "node@16"
```

Comment:
It does, but it's currently tested with node v16 and I'd prefer keep the same version everywhere. I'll make sure it's updated accordingly.

Reply:
Homebrew ships software with the newest dependencies that software works with. NodeJS 16 is well into it's maintenance period so we should really use something newer.

Reply:
I'll update the software to use node v18 then

Reply:
Why not 19?

Reply:
Because v18 is the active LTS, while v19 is just the future LTS (v20). See [nodejs.dev](https://nodejs.dev/en/about/releases/)

> Production applications should only use Active LTS or Maintenance LTS releases.

Reply:
Homebrew's policy is that any package should use the newest version that it is compatible with.